$Id: fix_createbox.patch 344 2012-06-29 15:38:56Z tg $

Fix CreateBox extension:
* XHTML/1.0 Transitional validity of output

--- a/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/CreateBox/CreateBox.php
+++ b/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/CreateBox/CreateBox.php
@@ -115,15 +115,13 @@ function acMakeBox( $input, $argv, $pars
 	$br = ( ( acGetOption( $input, 'break', 'no' ) == 'no' ) ? '' : '<br />' );
 	$label = acGetOption( $input, 'buttonlabel', wfMsgHtml( 'createbox-create' ) );
 	$output = <<<ENDFORM
-<div class="createbox" align="{$align}">
-<form name="createbox" action="{$submit}" method="get" class="createboxForm">
-<input type='hidden' name="action" value="create">
+<div class="createbox" align="{$align}"><form name="createbox" action="{$submit}" method="get" class="createboxForm">
+<input type='hidden' name="action" value="create" />
 <input type="hidden" name="prefix" value="{$prefix}" />
 <input type="hidden" name="preload" value="{$preload}" />
 <input type="hidden" name="editintro" value="{$editintro}" />
 <input class="createboxInput" name="title" type="text" value="{$text}" size="{$width}"/>{$br}
-<input type='submit' name="create" class="createboxButton" value="{$label}"/>
-</form></div>
+<input type='submit' name="create" class="createboxButton" value="{$label}"/></form></div>
 ENDFORM;
 	return $parser->replaceVariables( $output );
 }
